controllers/workspace_controller.go

```diff
@@ -641,5 +632,18 @@ func (r *WorkspaceReconciler) reconcileWorkspace(ctx context.Context, w *workspa
 	w.log.Info("Reconcile Runs", "msg", "successfully reconcilied runs")
 	r.Recorder.Eventf(&w.instance, corev1.EventTypeNormal, "ReconcileRuns", "Successfully reconcilied runs in workspace ID %s", w.instance.Status.WorkspaceID)
 
+	// Reconcile Outputs
+	// This reconciliation should always be after `reconcileRuns`
```

Comment:
Same observation as above
```suggestion
	// This reconciliation should always happen after `reconcileRuns`
```